utils/token_tracker.py

```
@@ -112,28 +249,169 @@ def get_instance(cls) -> 'TokenTracker':
+                    if len(self._unsent_records) > 200:
+                        self._unsent_records = self._unsent_records[-200:]
+                logger.debug(f"Token tracker: loaded {len(loaded_daily)} days of unsent telemetry from disk")
+            # 加载成功后删除文件，避免下次重复加载
+            p.unlink(missing_ok=True)
+        except Exception as e:
+            logger.debug(f"Token tracker: failed to load unsent queue: {e}")
+
+    def _save_unsent_queue(self):
+        """将当前未发送的远程数据持久化到磁盘。
+
+        调用时机：
```

Ruff (0.15.5)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)